src/main/resources/db/migration/V14__legg_til_rectype_i_statistikk_virksomhet_med_gradering.sql

```diff
@@ -0,0 +1 @@
+alter table sykefravar_statistikk_virksomhet_med_gradering add column rectype varchar(1);
```

Comment:
Disse migreringene kunne kanskje vært i samme fil?

Reply:
Synes det var ryddigere å ha de i to filer: 
 - en som gjelder endringen i struktur av tabellen (legg til en ny kolonne)
 - en som gjelder oppdatering av eksisterende data